lib/rubygems/specification.rb

```diff
@@ -2674,7 +2674,7 @@ def version=(version)
     end
     invalidate_memoized_attributes
 
-    return @version
+    return
```

Comment:
Like you did in bb7094a2d4246cf586ba55e1e0b645574936ab59, maybe better to completely remove this line. That would also allow us to enable https://docs.rubocop.org/rubocop/cops_style.html#styleredundantreturn.

Reply:
👍  I enabled `Style/RedundantReturn` cop.